File: Authorize/src/main/java/com/experiment/authorize/domain/base/BaseModel.java
package com.experiment.authorize.domain.base;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Data;

import java.util.Date;

@Data
public abstract class BaseModel {
    private Long id;
    @JsonIgnore
    private Long createdBy,updatedBy;
    private Date discontinueDate;
    @JsonIgnore
    private Date createdAt,updatedAt;
}

File: Inventory/src/main/kotlin/com/experiment/inventory/domain/base/BaseModel.kt
package com.experiment.inventory.domain.base

import com.fasterxml.jackson.annotation.JsonIgnore
import java.util.*
import javax.persistence.Column
import javax.persistence.Temporal
import javax.persistence.TemporalType

abstract class BaseModel {

    var id: Long? = null
    @JsonIgnore
    var createdBy: Long? = null
    @JsonIgnore
    var createdAt: Date? = null
    @JsonIgnore
    var updatedAt: Date? = null
    @JsonIgnore
    var updatedBy: Long? = null
    var discontinueDate: Date? = null
}